src/ralph/data_center/models/physical.py

```diff
@@ -164,6 +164,13 @@ class Rack(NamedMixin.NonUnique, models.Model):
     class Meta:
         unique_together = ('name', 'server_room')
 
+    def __str__(self):
+        return "{} ({}/{})".format(
+            self.name,
+            self.server_room.data_center,
```

Comment:
shouldn't  this
`self.server_room.data_center,`
be
`self.server_room.data_center.name,`
?